Main.java

```diff
@@ -3,58 +3,51 @@
 import java.io.*;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.sql.*;
 import java.util.StringTokenizer;
+import com.company.dbconnection.DatabaseLink;
 
 public class Main {
 
-    static final String FILE_TYPE_TXT = ".txt";
-    static final String FILE_TYPE_JSON = ".json";
-    static final String REMOVE_SPECIAL_CHAR = "[^a-zA-Z0-9 -]";
-    static final String SINGLE_SPACE = "";
-    static String searchResult;
+    static String textFilePath = null;
+    static String searchKeyword = null;
 
     public static void main(String[] args) {
 
-        String textFilePath;
-        String searchKeyword;
-        String errorMessage;
-        if (args.length != 2) {
-            errorMessage = "Provide valid filepath and search keyword";
-            System.out.println(errorMessage);
-            Database.databaseOperation(null, null, searchResult = "Failure", errorMessage, 0);
+        if((args.length != 2) || (args[0] == null || args[0] == " " || args[0] == "") || (args[1] == null || args[1] == " " || args[1] == "")) {
+            Constants.errorMessage1 = "Both file path and Search word must contain some value";
+            System.out.println(Constants.errorMessage1);
+            DatabaseLink databaseLink = new DatabaseLink(textFilePath, searchKeyword, Constants.FAILURE ,Constants.errorMessage1, 0);
+            databaseLink.start();
             return;
         }
-        textFilePath = args[0];
-        searchKeyword = args[1];
-        File file = new File(textFilePath);
-        System.out.println("Processing...");
-        try {
-            if (isValidFile(file, textFilePath, searchKeyword)) {
+            textFilePath = args[0];
+            searchKeyword = args[1];
+            File file = new File(textFilePath);
+            System.out.println("Processing...");
+            if(isValidFile(file, textFilePath, searchKeyword)) {
                 if (file.length() != 0) {
                     wordCount(textFilePath, searchKeyword);
                 } else {
-                    errorMessage = "File does not contain any data";
-                    System.out.println(errorMessage);
-                    Database.databaseOperation(textFilePath, searchKeyword, searchResult = "Failure", errorMessage, 0);
+                    Constants.errorMessage2 = "File does not contain any data";

```

Comment:
This error message should also be a constant

Reply:
Already did.